tests/app/helpers.py

```diff
@@ -79,36 +83,22 @@ def setup_authorization(app):
         HTTP_AUTHORIZATION='Bearer {}'.format(app.config['DM_SEARCH_API_AUTH_TOKENS']))
 
 
-def make_standard_service(**kwargs):
-    service = {
-        "id": "id",
-        "lot": "LoT",
-        "serviceName": "serviceName",
-        "serviceDescription": "serviceDescription",
-        "serviceBenefits": "serviceBenefits",
-        "serviceFeatures": "serviceFeatures",
-        "serviceCategories": ["serviceCategories"],
-        "supplierName": "Supplier Name",
-        "publicSectorNetworksTypes": ["PSN", "PNN"],
-    }
-
-    service.update(kwargs)
-
-    return {
-        "document": service
-    }
-
-
 def make_search_api_url(data, type_name='services'):
     # currently, data may contain either "document" or "service" (as a backward compatibility shim)
-    return '/index-to-create/{}/{}'.format(type_name, str(data.get('document', data.get('service'))['id']))
+    return '/test-index/{}/{}'.format(type_name, str(data.get('document', data.get('service'))['id']))
 
 
 def assert_response_status(response, expected_status):
```

Comment:
I'd be more in favour of getting rid of this method all together. No need to obscure this logic IMO. However, that `__tracebackhide__` is nifty

Reply:
Actually, we should do this as another PR

Reply:
I thought about that but the assertion method is fairly useful